libs/ui/src/examples/Fetch/E6_RequestUseQuery.tsx

```diff
@@ -2,27 +2,51 @@
 /* eslint-disable @typescript-eslint/no-unused-vars */
 import { FC, useEffect, useState } from 'react'
 
-import { Box, HStack, Select } from '@chakra-ui/react'
+import {
+  Alert,
+  AlertIcon,
+  Box,
+  Button,
+  Code,
+  FormControl,
+  HStack,
+  Input,
+  Select,
+  Spinner,
+} from '@chakra-ui/react'
 import { useQuery } from '@tanstack/react-query'
-import { StrapiLocale, StrapiUrl } from '@wsvvrijheid/types'
-import { request } from '@wsvvrijheid/utils'
+import { API_URL } from '@wsvvrijheid/config'
+import { Blog, StrapiLocale, StrapiModel, StrapiUrl } from '@wsvvrijheid/types'
+import { Request } from '@wsvvrijheid/utils'
+import axios from 'axios'
 
 import { LanguageSwitcher } from '../../admin'
 import { URL_OPTIONS } from './RequestPopulate'
 
 export const RequestUseQuery = () => {
   // TODO: Specify the type of all the states
-  const [url, setUrl] = useState('api/blogs')
-  const [filterValue, setFilterValue] = useState('')
-  const [filterProperty, setFilterProperty] = useState('title')
-  const [locale, setLocale] = useState('en')
+  const [url, setUrl] = useState<StrapiUrl>('api/blogs')
+  const [filterValue, setFilterValue] = useState<string>('Dimitri-2')
+  const [filterProperty, setFilterProperty] = useState<string>('title')
+  const [locale, setLocale] = useState<StrapiLocale>('tr')
 
   // Example filter: { [filterProperty]: filterValue }
   // eg { title: 'test' } for blogs
   // eg { name_${locale}: 'test' } for categories
 
   const fetchData = async () => {
     // TODO: Add request to fetch data
+
+    // eslint-disable-next-line prettier/prettier
+    const data = await Request.collection<StrapiModel[]>({
+      url: url,
+      filters: {
+        [filterProperty]: filterValue,
+      },
```

Comment:
It's easier to test with `containsi`

```suggestion
      filters: {
        [filterProperty]: { $containsi: filterValue },
      },
```

Reply:
thanks